Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -1322,6 +1451,18 @@
 				</dict>
 			</array>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>1.4</string>
+			<key>pfm_description</key>
+			<string>String to determine the path of an executable that is run when a new user is created. It is expected to return the UID that the user should be created with. It passes in the new users username as the first argument, default is to use the built in functionality</string>
+			<key>pfm_name</key>
+			<string>UIDTool</string>
+			<key>pfm_title</key>
+			<string>UIDTool</string>
```

Comment:
`UID Tool`?

Reply:
This is what Johan provided:

Here is the official documentation I'll be adding to the readme for it:
* UIDTool a String to determine the path of an executable that is run when a new user is created. It is expected to return the UID that the user should be created with. It passes in the new users username as the first argument, default is to use the built in functionality

Reply:
No problem with what it does, I just thought the pfm_title should have a space

Reply:
Ahhh sorry. I had removed it. But I re-added it, and added the space